cuda/runtime/BUILD.local_cuda

```diff
@@ -473,3 +473,67 @@ cc_library(
         ":nvtx_lib",
     ]),
 )
+
+# NPP
+_NPP_LIBS = {
+    "nppc": [
+        "npp.h",
+        "nppcore.h",
+        "nppdefs.h",
+    ],
+    "nppial": ["nppi_arithmetic_and_logical_operations.h"],
+    "nppicc": ["nppi_color_conversion.h"],
+    "nppidei": ["nppi_data_exchange_and_initialization.h"],
+    "nppif": ["nppi_filtering_functions.h"],
+    "nppig": ["nppi_geometry_transforms.h"],
+    "nppim": ["nppi_morphological_operations.h"],
+    "nppist": [
+        "nppi_statistics_functions.h",
+        "nppi_linear_transforms.h",
+    ],
+    "nppisu": ["nppi_support_functions.h"],
+    "nppitc": ["nppi_threshold_and_compare_operations.h"],
+    "npps": [
+        "npps_arithmetic_and_logical_operations.h",
+        "npps_conversion_functions.h",
+        "npps_filtering_functions.h",
+        "npps.h",
+        "npps_initialization.h",
+        "npps_statistics_functions.h",
+        "npps_support_functions.h",
+    ],
+}
+
+[
+    cc_import(
+        name = name + "_so",
+        shared_library = "cuda/lib64/lib{}.so".format(name),
+        target_compatible_with = ["@platforms//os:linux"],
+    )
+    for name in _NPP_LIBS.keys()
+]
+
+[
+    cc_import(
+        name = name + "_lib",
+        interface_library = "cuda/lib/x64/nppc.lib",
```

Comment:
Should `nppc.lib` a mistake of `{}.lib` to be formatted with `name`?

Reply:
Absolutely, thanks for the catch. I was not able to test this on Windows as you might have guessed ;P